tests/jvm/src/test/scala/scala/meta/tests/semanticdb/TargetedSuite.scala

```diff
@@ -161,8 +162,6 @@ class TargetedSuite extends DatabaseSuite(SemanticdbMode.Slim) {
        |_root_.f.C1#T1# => abstract type T1
        |_root_.f.C1#T2# => type T2: Int
        |  [0..3): Int => _root_.scala.Int#
-       |_root_.f.C1#`<init>`()V. => secondaryctor <init>: (): C1
-       |  [4..6): C1 => _root_.f.C1#
```

Comment:
Can you also mark this and other constructor-related issues as TODO?

Reply:
I would love to accomplish this refactoring with 0 breaking changes, and I believe it's possible without scalameta parse step but it requires more time to get right.